phpstan.neon

```diff
@@ -423,3 +423,23 @@ parameters:
             message: '#"%s" in sprintf\(\) format must be quoted#'
             paths:
                  - packages/monorepo-builder/packages/release/src/Command/ReleaseCommand.php #110
+
+        -
+            message: '#Content of method ".*" is duplicated with method in ".*" class, use unique content instead#'
+            paths:
+                 - packages/php-config-printer/src/DependencyInjection/Extension/PhpConfigPrinterExtension.php #14
+                 - packages/phpstan-rules/src/Rules/PreventDuplicateClassMethodRule.php #73
+                 - packages/skipper/src/DependencyInjection/Extension/SkipperExtension.php #14
+                 - packages/smart-file-system/src/Json/JsonFileSystem.php #33
+                 - packages/smart-file-system/src/Json/JsonFileSystem.php #41
+                 - packages/easy-coding-standard/src/DependencyInjection/Extension/EasyCodingStandardExtension.php #14
+                 - packages/console-package-builder/tests/HttpKernel/ConsolePackageBuilderKernel.php #24
+                 - packages/coding-standard/src/TokenRunner/Analyzer/FixerAnalyzer/BlockFinder.php #107
+                 - packages/phpstan-extensions/src/ReturnTypeExtension/ContainerGetReturnTypeExtension.php #52
+                 - packages/rule-doc-generator/src/RuleCodeSamplePrinter/RectorRuleCodeSamplePrinter.php #73
+                 - packages/latte-to-twig-converter/src/Contract/CaseConverter/CaseConverterInterface.php #14
```

Comment:
Interfaces should be skipped, as they're empty

Reply:
implemented.